packages/rich-text/src/to-dom.js

```diff
@@ -58,9 +58,28 @@ function getNodeByPath( node, path ) {
 	};
 }
 
+/**
+ * Returns a new instance of a DOM tree upon which RichText operations can be
+ * applied.
+ *
+ * Note: The current implementation will return a shared reference, reset on
+ * each call to `createEmpty`. Therefore, you should not hold a reference to
+ * the value to operate upon asynchronously, as it may have unexpected results.
+ *
+ * @return {WPRichTextTree} RichText tree.
+ */
 function createEmpty() {
```

Comment:
There's also `createElement`, see #12470. It would be great if we could merge these two functions into one?

Reply:
Maybe we can update the PR here to also address `createElement` and only keep one.